src/globals.h

```diff
@@ -906,9 +907,11 @@ struct Warp_t
     bool noPrintStars = false;
     bool noEntranceScene = false;
     bool cannonExit = false;
+    bool stoodRequired = false; // Require player stood on the ground to enter this warp
     double cannonExitSpeed = 10.0;
     std::string eventEnter;
     std::string StarsMsg;
+    int transitEffect = 0;
```

Comment:
Use an enum instead? :)

Reply:
It's already enum just an integer container here (to don't include PGE-FL structures here at "globals.h"), in the code you'll find direct enum usage :wink:

Reply:
Ah right, I usually use C++-style enum classes so I was expecting to see that. This is good too! :)